rss2email/post_process/redirect.py

```diff
@@ -61,7 +61,7 @@ def process(feed, parsed, entry, guid, message):
     for link in links:
         try:
             request = urllib.request.Request(link)
-            request.add_header('User-agent', rss2email.feed._USER_AGENT)
+            request.add_header('User-agent', feed.user_agent)
```

Comment:
We don’t hit this line with our tests?

Reply:
`git grep redirect` in the `test` directory returns no matches, so it does seem that way.